### ISSUE TITLE
PostBuffer not working since 0.4.7

### ISSUE THREAD

**Issue by Marsup** [dodo/node-bufferstream#5]
Hello !

I have been using bufferstream since a few months and I've noticed that versions > 0.4.6 give me an empty buffer, I haven't identified the culprit (but may try to if I have time), so if you see anything wrong before I do, it will be much appreciated :)

Regards.

**Comment by dodo** [comment]
could you please give me a short code part how you use it?

**Comment by Marsup** [comment]
Here is a minimal code to test it :

``` javascript
var http = require('http'),
    PostBuffer = require('bufferstream/postbuffer');

http.createServer(function(req, res) {
  var postHandler = new PostBuffer(req);
  postHandler.onEnd(function(data) {
    console.log('Data:', data.toString());
  });
}).listen(6000, function ready() {
  var request = http.request({ port: 6000, method: 'POST' });
  request.end(new Buffer('teststring'));
});
```

**Comment by dodo** [comment]
fixed in [eaf33ef](https://github.com/dodo/node-bufferstream/commit/eaf33effc2ec3ac1aab3fbaf0489ec1a1a9dea45).

thx for reporting this.

**Comment by Marsup** [comment]
Fix confirmed, thanks a lot !
